_data/jobs.yml

```diff
@@ -1,3 +1,23 @@
+- expires: 2022-02-28
+  location: Allen Institute, Seattle, WA
+  name: Data Analyst I
+  posted: 2021-12-10
+  url: https://alleninstitute.hrmdirect.com/employment/job-opening.php?req=1799205&&cust_sort1=29268&&nohd#job
+- expires: 2022-01-10
+  location: Lawrence Berkeley National Laboratory, Berkeley, CA
+  name: Software and User Interface Engineer
+  posted: 2021-12-10
+  url: https://lbl.referrals.selectminds.com/jobs/software-and-user-interface-engineer-4473
+- expires: 2022-01-10
+  location: Lawrence Berkeley National Laboratory, Berkeley, CA
+  name: Software Engineer for Scientific Data Systems
+  posted: 2021-12-10
+  url: http://50.73.55.13/counter.php?id=215965
```

Comment:
@sandragesing did you check this URL? I'm wincing that I didn't - counter.php (or something that looks like it) is well known Malware that redirects a clicker to a malicious site. The domain is now 404 (and I saw and removed it) but I'm worried we might have received a job post from a malicious entity just copying another job title / location. I would expect an institution like Berkeley to not have a temporary server with http (and not https) and to not have a name like counter.php (e.g., see https://threatpost.com/counter-php-found-redirecting-to-sites-peddling-styx-exploit-kit/101967/)